apps/website/docusaurus.config.js

```diff
@@ -57,29 +64,46 @@ const config = {
           autoCollapseCategories: false,
         },
       },
+      footer: {
+        links: [
+          {
+            label: 'Open source at Microsoft',
+            href: 'https://opensource.microsoft.com',
+          },
+          {
+            label: 'Microsoft Privacy Statement',
+            href: 'http://aka.ms/privacy',
+          },
+        ],
+      },
       navbar: {
-        title: 'Griffel',
-        // logo: {
-        //   alt: 'My Site Logo',
-        //   src: 'img/logo.svg',
-        // },
+        logo: {
+          alt: 'Griffel.js',
+          src: 'img/griffel.png',
+        },
         items: [
           {
             type: 'doc',
             docId: 'react/install',
-            position: 'left',
+            position: 'right',
             label: 'For React.js',
           },
           {
             type: 'doc',
             docId: 'try-it-out/try-it-out',
-            position: 'left',
+            position: 'right',
             label: 'Try it out',
           },
+          {
+            href: 'https://github.com/facebook/docusaurus',
+            position: 'right',
+            className: 'navbar-github-link',
+            html: 'Github <span aria-hidden="true" />',
+          },
```

Comment:
Replaced custom `Navbar/ColorModeToggle`, so that component is no longer ejected 💪